introduction.md

```diff
@@ -1 +1,6 @@
 Hey, I'm Dan from London and currently learning git. My favourite Code School path at the moment is Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?